communication/slack_channels.md

```diff
@@ -0,0 +1,62 @@
+# Slack Channels
+
+Please find a list here of Slack channels at source{d}.
+
+Slack channels are encouraged to be open for everyone to join. Private channels for professional reasons are highly discouraged (have fun with private personal slack channels).
+
+### General channels
+
+* `#general`
+* `#offtopic`
+* `#officechat`
+  * Physical office related discussions
```

Comment:
Should this be Madrid office-only or relative to SF as well?

Reply:
Both offices, I'll make it clear.